BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py

```diff
@@ -55,85 +57,147 @@
+                    
+                    fileName = os.path.join(path, name)
+                    IceFinalFile = ROOT.TFile(fileName)
+                    
                     passTree = IceFinalFile.passTree
                     passEvents = passTree.GetEntries()
+                    
+                    nuWeights = []
+                    nuPasses = []
+                    print(passEvents)
+                    
                 except:
                     print("Error opening file")
                     continue
-                success_runs.append(fileName)
+                
+                
+
+                passTree.GetEvent(0)
+                #Create a canvas for the TGraphs
+                c = ROOT.TCanvas("c", "c", 1000, 1000)
+                c.Divide(1, 2)
+                c.cd(1)
+                signalAtDetector = passTree.event.signalAtDetector
+                # Add titles and axes to the TGraph and then save as a png
+                signalAtDetector.SetTitle("Signal at Detector for {} EeV Neutrinos".format(g.energy))
+                signalAtDetector.GetXaxis().SetTitle("Time (ns)")
+                signalAtDetector.GetYaxis().SetTitle("Voltage (V)")
+                signalAtDetector.SetLineColor(1)
+                signalAtDetector.SetLineWidth(2)
+                signalAtDetector.SetMarkerStyle(20)
+                signalAtDetector.SetMarkerSize(0.5)
+                signalAtDetector.SetMarkerColor(1)
+                signalAtDetector.Draw()
+                c.cd(2)
+                signalAtSource = passTree.event.signalAt1m
+                signalAtSource.SetTitle("Signal at 1m for {} EeV Neutrinos".format(g.energy))
+                signalAtSource.GetXaxis().SetTitle("Time (ns)")
+                signalAtSource.GetYaxis().SetTitle("Voltage (V)")
+                signalAtSource.SetLineColor(1)
+                signalAtSource.SetLineWidth(2)
+                signalAtSource.SetMarkerStyle(20)
+                signalAtSource.SetMarkerSize(0.5)
+                signalAtSource.SetMarkerColor(1)
+                signalAtSource.Draw()
+                c.Print("{}/signalPlots/{}_{}_signals_bestindiv.png".format(g.destination, g.indiv, counter))
+                counter += 1
+                c.Close()
+                
                 
                 for i in range(passEvents):
                     j += 1
                     
                     passTree.GetEvent(i)
-                    allTree.GetEvent(i)
-                    
+                
                     # Part from rootAnalysis.py
                     nuPasses.append(1)
                     nuWeights.append(passTree.event.neutrino.path.weight
-                                     /(passTree.event.loop.positionWeight
-                                       *passTree.event.loop.directionWeight))
+                                    /(passTree.event.loop.positionWeight
+                                    *passTree.event.loop.directionWeight))
```

Comment:
The previous indentation was ok, even though I prefer to leave the operator at the end of the previous line and align the args:
```py
                    nuWeights.append(passTree.event.neutrino.path.weight /
                                     (passTree.event.loop.positionWeight *
                                      passTree.event.loop.directionWeight))